src/routes/event/index.tsx

```diff
@@ -66,129 +66,146 @@ const Event = ({ eventKey }: Props) => (
-                        href: gmapsUrl(eventInfo.lat, eventInfo.lon),
-                        target: '_blank',
-                        rel: 'noopener',
-                        action: <Icon icon={googleMaps} />,
-                      },
-                      eventInfo.district && {
-                        icon: infoOutline,
-                        title: (
-                          <span>
-                            {
-                              (eventInfo as { fullDistrict: string })
-                                .fullDistrict
-                            }{' '}
-                            <Chip>{eventInfo.district}</Chip>
-                          </span>
-                        ),
-                      },
-                      {
-                        icon: calendar,
-                        title: (
-                          <span>
-                            {formatDateRange(
-                              eventInfo.startDate,
-                              eventInfo.endDate,
-                            )}{' '}
-                            {eventInfo.week !== undefined && (
-                              <Chip>{`Wk ${eventInfo.week + 1}`}</Chip>
-                            )}
-                          </span>
-                        ),
-                        action: (
-                          <IconButton
-                            href={gcalUrl(eventInfo)}
-                            target="_blank"
-                            rel="noopener"
-                            icon={calendarPlus}
-                          />
-                        ),
-                      },
-                      eventInfo.webcasts.length > 0 && {
-                        icon: video,
-                        title:
-                          'Live Stream' +
-                          (eventInfo.webcasts.length === 1 ? '' : 's'),
-                        href:
-                          eventInfo.webcasts.length === 1
-                            ? eventInfo.webcasts[0]
-                            : undefined,
-                        target: '_blank',
-                        rel: 'noopener',
-                        action:
-                          eventInfo.webcasts.length === 1 ? (
-                            <Icon icon={webcastIcon(eventInfo.webcasts[0])} />
-                          ) : (
-                            eventInfo.webcasts.map(w => (
-                              <IconButton
-                                key={w}
-                                icon={webcastIcon(w)}
-                                href={w}
-                                target="_blank"
-                                rel="noopener"
-                              />
-                            ))
+    renderSuccess={({ matches, eventInfo, eventStats, schema }) => {
+      const now = new Date()
+      const nextMatch = matches
+        ? matches.find(m => m.time !== undefined && m.time > now) || matches[0]
```

Comment:
It should not display a "next match" if all matches are in the past
I don't think we need a `|| matches[0]`